src/psyclone/psyir/backend/sir.py

```diff
@@ -113,6 +113,10 @@ def __init__(self, skip_nodes=False, indent_string="  ",
                  initial_indent_depth=0):
         super(SIRWriter, self).__init__(skip_nodes, indent_string,
                                         initial_indent_depth)
+        # The _field_names variable stores the unique field names
+        # found in the PSyIR. This is required as the SIR declares
+        # field names after the computation using the fields has been
```

Comment:
Could just end with "after the computation." Not sure what the last bit is saying.

Reply:
Changed.